networkit/cpp/community/AdjustedRandMeasure.cpp

```diff
@@ -38,7 +38,7 @@ double NetworKit::AdjustedRandMeasure::getDissimilarity(const NetworKit::Graph &
         sumEta += s * (s - 1) / 2;
     }
 
-    count n = G.numberOfNodes();
+    const double n = G.numberOfNodes();
```

Comment:
Same here.

Reply:
Here `n` is used to compute `double expectedIndex = sumZeta * sumEta / (n * (n-1) / 2);` isn't the denominator computed as an integer?

Reply:
Yes, but the argument is the same as above: `n * (n-1)` is always even, so the divison never truncates.

Reply:
Which is the same argument as betweenness, ok I will change this.